opt/build/libtorch/repack.sh

```diff
@@ -24,8 +24,7 @@ ARCH=$(python3 $ROOT/opt/readies/bin/platform --arch)
 [[ $OS == macosx ]] && export LC_ALL=en_US.UTF-8
 
 if [[ -z $PT_VERSION ]]; then
-	PT_VERSION=1.2.0
-	#PT_VERSION="latest"
+	PT_VERSION="latest"
```

Comment:
Just a confirmation: if `PT_VERSION` is set in `get_deps` (that is, if repack is called from there), then we don't set it to `latest` right?

Reply:
That's right. We might as well bail out with an error, as we never use this script for a "floating" version.